### PR TITLE
Release 4.0.9

### PR DESCRIPTION
**Pull request by mfussenegger**
## Checklist

 - [x] User relevant changes are recorded in ``CHANGES.txt``
 - [x] Touched code is covered by tests
 - [x] Documentation has been updated if necessary
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [x] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

File: docs/appendices/release-notes/4.0.9.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.. _version_4.0.9:
+
+=============
+Version 4.0.9
+=============
+
+Released on 2019/11/07.
```
Comment (marregui) [inline docs/appendices/release-notes/4.0.9.rst]: Is that date in the past in purpose?

File: es/es-server/src/main/java/org/elasticsearch/Version.java [hunk 2]
```diff
@@ -86,7 +86,7 @@
     public static final Version V_4_0_6 = new Version(ES_V_7_0_6_ID, ES_V_7_0_6_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
     public static final Version V_4_0_7 = new Version(ES_V_7_0_7_ID, ES_V_7_0_7_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
     public static final Version V_4_0_8 = new Version(ES_V_7_0_8_ID, ES_V_7_0_8_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
-    public static final Version V_4_0_9 = new Version(ES_V_7_0_9_ID, ES_V_7_0_9_ID - INTERNAL_OFFSET, true, org.apache.lucene.util.Version.LUCENE_8_0_0);
+    public static final Version V_4_0_9 = new Version(ES_V_7_0_9_ID, ES_V_7_0_9_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
```
Comment (marregui) [inline es/es-server/src/main/java/org/elasticsearch/Version.java]: Do we need to create at this point V_4_0_10 snapshot?

Comment (mfussenegger) [reply es/es-server/src/main/java/org/elasticsearch/Version.java]: That will be a separate commit

File: docs/appendices/release-notes/4.0.9.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/4.0.9.rst]: No, copy & paste mistake. Fixed it.

**Pull request merged**